<launch>
    <rosparam command="load" file="$(find black_car)/settings.yaml"/>
    <node pkg="black_car" name="follow_the_gap" type="follow_the_gap" output="screen">
        <remap from="/drive" to="/vesc/high_level/ackermann_cmd_mux/input/nav_0"/>
    
    </node>
</launch>
